src/clj/rems/db/test_data.clj

```diff
@@ -859,10 +868,16 @@
 
         ;; Create resources
         res1 (create-resource! {:resource-ext-id "urn:nbn:fi:lb-201403262"
-                                :organization {:organization/id "nbn"}
+                                :organization {:organization/id "nbn"
+                                               :organization/short-name {:en "NBN en"
+                                                                         :fi "NBN fi"
+                                                                         :sv "NBN sv"}}
```

Comment:
`create-resource!` only cares about the `organization/id`. The other members of `:organization` are ignored. You can verify this by following the definitions to `rems.api.services.resource/create-resource!` if you want.

To explain that a bit, `create-resource!` only wants a reference to an existing organization, via a parameter like `:organization {:organization/id "foo"}`. It doesn't create the organization. The `:organization/short-name` is only needed when calling `create-organization!`.

Reply:
I ll fix this, I am pretty sure you mentioned it last time but it somehow escaped my memory